controllers/workoutlog.js

```diff
@@ -1,3 +1,19 @@
+<<<<<<< HEAD
+const Log = require('../models/workoutlog');
+
+module.exports = {
+  index, 
+}; 
+
+
+function index(req, res) {
+  Log.find({}, function (err, workoutlog) {
+    res.render('user/index'); 
+  })
```

Comment:
Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7840230)
```suggestion
} 
```